plugins/inputs/aerospike/aerospike.go

```diff
@@ -121,6 +121,8 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 func parseValue(v string) interface{} {
 	if parsed, err := strconv.ParseInt(v, 10, 64); err == nil {
 		return parsed
+	} else if parsed, err := strconv.ParseUint(v, 10, 64); err == nil {
+		return -int64(parsed)
```

Comment:
Quite obvious there is a bug on Aerospike server's side rendering this metric unusable in some cases. Situation of `data_used_bytes_memory` becoming negative is not quite correct itself, not speaking of this uint <-> int conversion inside Aerospike.

We need a way to handle large uint64() without affecting other Aerospike metrics submission, since influxdb rejects the whole payload when it gets `data_used_bytes_memory="18446744041841121751"` for a metric that used to be a normal int metric like `data_used_bytes_memory=42i`.

I'm not sure if it's possible to handle uint64's like a number since there is no good way to store them at least in InfluxDB, other backends most likely are going to have similar problem.

Perhaps it's better to leave result coming from Aerospike as negative integer, without reversing it.

What do you think?